### PR TITLE
correct return code for get_remote_files (always returned 0 before)

### PR DESCRIPTION
**Pull request by ddbox**

### CODE REVIEW

File: src/tests/fixtures/mock_imports.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import mock
```
Comment (renbauer-fnal) [inline src/tests/fixtures/mock_imports.py:1]: Is this file still needed? I think with the correct Python version (like the one installed on our pytest image) we should be able to import these modules without issue.

File: src/tests/fixtures/enrcp [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (renbauer-fnal) [inline src/tests/fixtures/enrcp:1]: This might also be unneeded on our test image due to Enstore package installation, but I'm not sure

File: src/tests/fixtures/mock_imports.py (hunk 1 above)
Comment (ddbox) [reply src/tests/fixtures/mock_imports.py:1]: Its not needed, but its a convenience when I am working on my laptop at home.  Arguably I could be running the tests in a container on my laptop at home

**Comment by ddbox** [comment]
I am going to close this pull request as the changes were already merged in PR49
